### DISCUSSION TITLE
Feature Request: Map and Line of sight tool

### DISCUSSION THREAD

**Discussion by koposcabana** [Lurkars/gloomhavensecretariat#86] [Ideas]
Requested Reimplementation within tool of something similar too: https://gloomhaven.one/

If you also created a "creator's" portal, where users/devs could submit maps that would be awesome!

Requested for the following reasons:
1. Tool elimination. I'd love to just pull up the map within the tool and line of sight.
2. The new automatic actions based on room tiles are awesome. I however don't utilize room tiles and have 3d printed terrain. I have to then pull out the book and look at the room tile names to confirm correct action.

**Comment by Lurkars** [comment]
Okay, I can just tell that this will out of scope.

The only integration of such tools I can think of, is to just link them somewhere. Because the wording is exactly where I see a difference (which of course will be difficult to determine in detail, but:) GHS is a COMPANION app, so it should accompany you every round/scenario/campaign , where a TOOL is something you only need in certain situation to help you (to apply rules correctly etc). And I don't see implementing tools into GHS (except the internal tools for GHS itself like the data editors).

Besides all that I am still very busy with getting everything done for existing editions, the new room/rules stuff is missing for most editions than GH and JOTL. There is also Trail of Ashes which will be asked soon. So only the content for the current features is still a lot of work to do. And if you take a look at the long term feature, it's character's ability cards and together with implementation of view and useful integration being a lot of work, the data is nowhere available in a technical format, so creating all ability decks will take month afterwards.
